depoly.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+# Fist generate code context for the cloud debugger
+gcloud debug source gen-repo-info-file
+for module in src/*; do
+    cp source-context.json "$module"
+done
+
+# Now deploy on GKE
+skaffold run
```

Comment:
We don't want to hide skaffold. Please avoid creating a deployment script. A major part of the demo is to show direct deployment with skaffold.